1.10/installing/custom/configuration/configuration-parameters.md

```diff
@@ -418,22 +419,30 @@ A YAML nested list (`-`) of DNS resolvers for your DC/OS cluster nodes. You can
 **Caution:** If you set the `resolvers` parameter incorrectly, you will permanently damage your configuration and have to reinstall DC/OS.
 
 ### rexray_config
-The <a href="https://rexray.readthedocs.org/en/v0.3.2/user-guide/config/" target="_blank">REX-Ray</a> configuration method for enabling external persistent volumes in Marathon. REX-Ray is a storage orchestration engine. The following is an example configuration.
-
-    rexray_config:
-      rexray:
-        loglevel: info
-        modules:
-          default-admin:
-            host: tcp://127.0.0.1:61003
-        storageDrivers:
-        - ec2
+The <a href="https://rexray.readthedocs.org/en/v0.3.2/user-guide/config/" target="_blank">REX-Ray</a> configuration for enabling external persistent volumes in Marathon. REX-Ray is a storage orchestration engine. The following is an example configuration.
+
+    rexray: 
+      loglevel: info 
+      service: ebs
+    libstorage:
+      integration:
         volume:
-          unmount:
-            ignoreusedcount: true
+          operations:
+            unmount:
+              ignoreusedcount: true
+      server:
+        tasks:
+          logTimeout: 5m
 
 See the external persistent volumes [documentation](/docs/1.10/storage/external-storage/) for information on how to create your configuration.
 
+If the `rexray_config` parameter is provided, its contents are used verbatim for REX-Ray's configuration. This lets you define completely custom REX-Ray configurations which integrate with various [external storage providers](https://rexray.readthedocs.io/en/stable/user-guide/storage-providers/). However, if you upgrade your cluster to a version that includes an updated version of REX-Ray, you must ensure that your `rexray_config` parameter is compatible with the newer version of REX-Ray.
```

Comment:
I know I provided the original link to `stable` REX-Ray, but what do you think about being specific about the version (ie., https://rexray.readthedocs.io/en/v0.9.0/user-guide/storage-providers/ )